code/controllers/subsystems/inactivity.dm

```diff
@@ -34,6 +33,9 @@ SUBSYSTEM_DEF(inactivity)
 					if(job)
 						information = " while [job]."
 
+				else if(isobserver(C.mob))
+					information = "while a ghost."
```

Comment:
Might need a space before the `while`, to be consistent with the other messages.